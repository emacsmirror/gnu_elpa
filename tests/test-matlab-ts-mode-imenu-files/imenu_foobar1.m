% -*- matlab-ts -*-
function imenu_foobar1(a, b, c)
end 

function foobar2(a, b, c)
end 

function gen_pulse_avg_lin_data(a, b, c)
end

function gen_pulse_avg_log_data(a, b, c)
end

function gen_beamsharpened_data(a, b, c)
end

function foobar3(a, b, c)
end
