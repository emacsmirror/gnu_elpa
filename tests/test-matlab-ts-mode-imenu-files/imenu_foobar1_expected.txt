foobar3
gen_beamsharpened_data
gen_pulse_avg_log_data
gen_pulse_avg_lin_data
foobar2
imenu_foobar1
